startup.sh

```diff
@@ -1,4 +1,13 @@
 #!/bin/bash
 
 bundle install
-bundle exec rails s -p 3065
+
+if [[ $1 == "--live" ]] ; then
+  GOVUK_APP_DOMAIN=www.gov.uk \
+  GOVUK_WEBSITE_ROOT=https://www.gov.uk \
+  PLEK_SERVICE_CONTENT_STORE_URI=https://www.gov.uk/api \
+  PLEK_SERVICE_STATIC_URI=assets.publishing.service.gov.uk \
+  bundle exec rails s -p 3065
+else
+  bundle exec rails s -p 3065
+fi
```

Comment:
@cbaines i had a try with that, but i could not get that working, due to them not being exports, and then you cant copy and paste without ruining your own env, if you know how to do it, i'm happy to put it in.